docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       start_period: 30s
 
   graphql-engine:
-    image: onfinality/subql-query:v1.0.0
+    image: onfinality/subql-query:latest
```

Comment:
imo its better to use an explicit version here so it won't break all of a sudden on for someone + we'll be testing with the same versions